syntax = "proto3";

package lmjwtest;

// service, encode a plain text 
service EncodeService {
    // request a service of encode
    rpc GetEncode(plaintext) returns (encodetext) {}
}


message plaintext {
    label label1 = 4;
    textxxx text1 = 5;
    string pttransactionID = 1;
    string ptproperties = 2;
    string ptsenderID = 3;
}

message label{
    string labeltext = 1;
}

message textxxx{
    string textlabel = 1;
}

message encodetext {
    string enctransactionID = 1;
    string encproperties = 2;
    string encsenderID = 3;
}
